java/sbt/oss/build.sbt

```diff
@@ -7,9 +7,12 @@ crossPaths := false
 autoScalaLibrary := false
 
 // Please make sure that following dependencies have versions corresponding to the ones in your cluster.
+// Note that spark-hive and spark-cassandra-connector should be provided with '--packages' flag to
+// spark-submit command.
 libraryDependencies ++= Seq(
-  "org.apache.spark" %% "spark-core" % "1.6.1" % "provided",
-  "org.apache.spark" %% "spark-sql" % "1.6.1" % "provided",
+  "org.apache.spark" %% "spark-core" % "1.6.2" % "provided",
```

Comment:
not super important but we could always have sparkVersion as a variable to avoid repeating